defect-tracker-server/src/main/java/com/defect/tracker/util/ValidationFailureStatusCodes.java

```diff
@@ -58,7 +58,7 @@ public void setProjectempAlreadyExists(String projectempAlreadyExists) {
 		this.projectempAlreadyExists = projectempAlreadyExists;
 	}
 	
-	@Value("${validation.project.getProjectempNotExist}")
+	@Value("{validation.project.getProjectempNotExist}")
```

Comment:
$ deleted?

Reply:
I already put this $ in earlier with my branch in yesterday  evening  so 2nd time,  I  removed  that, it's  fixed for PullRequest in Endpoint URI and Validation  Constance changes named  commited